lib/osf-components/package.json

```diff
@@ -16,8 +16,7 @@
     "ember-cp-validations": "*",
     "ember-css-modules": "*",
     "ember-css-modules-sass": "*",
-    "ember-i18n": "*",
-    "ember-i18n-inject": "*",
```

Comment:
Talked with @jamescdavis and we don't want to replicate this injection, which means we'll likely need to add ember-intl imports to quite a few places in code that don't have them yet.